template.json

```diff
@@ -10,41 +10,77 @@
             "background": {
                 "biography": ""
             },
-            "resources": {
-                "hitPoints": {
-                    "min": 0,
-                    "value": 10,
-                    "max": 40
-                },
-                "pietas": {
-                    "min": 0,
-                    "value": 10,
-                    "max": 50
+            "common": {
+                "attributes":
+                {
+                    "hitPoints": {
+                        "min": 0,
+                        "value": 10,
+                        "max": 40
+                    },
+                    "pietas": {
+                        "min": 0,
+                        "value": 10,
+                        "max": 50
+                    }
                 }
             }
         },
         "custos": {
             "templates": [
                 "background",
-                "resources"
+                "common"
             ],
+            "peritiae":
+            {
+                "deBello":
+                {
+                    "value": 14,
+                    "defaultRoll": "d10+d4"
+                },
+                "deNatura":
+                {
+                    "value": 17,
+                    "defaultRoll": "d12+d5"
+                },
+                "deSocietate":
+                {
+                    "value": 6,
+                    "defaultRoll": "d6"
+                },
+                "deMagia":
+                {
+                    "value": 8,
+                    "defaultRoll": "d8"
+                },
+                "deScientia":
+                {
+                    "value": 6,
+                    "defaultRoll": "d6"
+                },
+                "deCorpore":
+                {
+                    "value": 14,
+                    "defaultRoll": "d10+d4"
+                }
+            },
             "FateDice": true
         },
         "friendly": {
             "templates": [
-                "resources"
+                "background",
+                "common"
             ]
         },
         "antagonist": {
             "templates": [
                 "background",
-                "resources"
+                "common"
             ]
         },
         "fantasticalCreature": {
             "templates": [
-                "background",
-                "resources"
+                "common"
             ],
```

Comment:
I do think more of these categories should have peritiae. In combat, the player rolls De Bello against the enemy De Corpore (or De Bello? not sure) thus the enemy should have such an attribute.